### DISCUSSION TITLE
Upgrade issue "claro_security_login" does not exist

### DISCUSSION THREAD

**Discussion by smallBCheung2020** [claroline/Claroline#1648] [Q&A] [answered]
I had upgrade the current older version of claroline to the latest one 13.0.12. After that, I cannot login to the system even using an admin account. It stuck in "login in progress".

I tried to replicate and it shows the following error message in the browser console:

Uncaught Error: The route "claro_security_login" does not exist.

I tried recompiling again following the steps mentioned in https://github.com/claroline/Claroline but the issue persists. In the meantime, I restored the previous version. However, I'm unable to add any file to the workspace.

Please help.

**Comment by LaurentGruber** [comment, marked as answer]
Hi,

The internet.domain_name and internet.platform_url are null in the files/config/platform_options.json file imo
Set it to the good domain name and the good url (with http:// or https://).
check also the ssl.enabled param if you are in https

For the adding file issue, check the permission on files folder it must be writable by the webserver
